packages/adena-extension/src/common/utils/client-utils.ts

```diff
@@ -9,6 +9,11 @@ import BigNumber from 'bignumber.js';
 import dayjs from 'dayjs';
 import fetchAdapter from '@vespaiach/axios-fetch-adapter';
 
+export function formatHash(hash: string, num?: number): string {
```

Comment:
looks like `num` isn't actually used.

do we need to keep this optional argument?

Reply:
Currently we don't use it.
However, we would like to match the form with other formatting functions. 
Does it look good to remove them ?

Reply:
nah let's keep it for now

we can get rid of it when it becomes unnecessary